src/resources/postcss/tickets-refresh.pcss

```diff
@@ -76,7 +76,7 @@
 	fieldset {
 		border: 0;
 		padding: 0;
-		margin: 0;
+		margin: 0 0 1em 0;
```

Comment:
Technically you don't need that final `0`. But you don't have to remove it in this PR.